utils/scarb-test-support/src/filesystem.rs

```diff
@@ -53,3 +55,10 @@ pub fn path_with_temp_dir(t: &TempDir) -> OsString {
     let other_paths = env::split_paths(&os_path);
     env::join_paths(script_path.chain(other_paths)).unwrap()
 }
+
+pub fn hash_path(path: &str) -> String {
+    let mut hasher = DefaultHasher::new();
```

Comment:
Note, that the DefaultHasher is PRNG-seeded, so to make the hashes consistent between runs you need to use `scarb_stable_hash::StableHasher`.